.github/workflows/snapshot-file-copy.yml

```diff
@@ -43,6 +43,7 @@ jobs:
       - name: 'GCS upload files'
         id: 'gcs-upload-file'
         uses: 'google-github-actions/upload-cloud-storage@a5b77a3bf84da1791719585d327e5f90ae5cb53c' # ratchet:google-github-actions/upload-cloud-storage@v1
+        if: '${{ hashFiles(format(''./{0}/*'', inputs.path)) }}'
```

Comment:
is it to check if files match pattern ./{0}/* exist? should it be '${{ hashFiles(format(''./{0}/*'', inputs.path)) != '''' }}'?

Reply:
Yes. I think this is a abbreviation of it.